anvil/core/Cargo.toml

```diff
@@ -28,4 +28,6 @@ reference-trie = { version = "0.25" }
 keccak-hasher = { version = "0.15" }
 
 [features]
-default = ["serde"]
+default = []
```

Comment:
If you want to set `fastrlp` and `serde` as default features, you should still keep their respective feature flags, such that people can do `{ ... , default-features = false, features = ["serde"] }`